Data Structure Session 6 Assignment/DataStructure6-Q2/src/com/metacube/dto/FindTotalUniqueCharacterDTO.java

```diff
@@ -0,0 +1,31 @@
+package com.metacube.dto;
+
+/**
+ * The Class UniqueCharacterDTO.
+ */
+public class FindTotalUniqueCharacterDTO {
+
+	/** if successful then true else false. */
+	public boolean success;
+
+	/** contains the result */
+	public int result;
+
+	/** true if data from cache else false */
+	public boolean fromCache;
+
+	/**
+	 * Check if objects are equals or not
+	 */
+	@Override
+	public boolean equals(Object obj) {
```

Comment:
This equals method is used for test cases only, So instead of writing this here, we can just create a method in test cases to perform comparison.